CHANGELOG.md

```diff
@@ -11,6 +11,10 @@
 
 - Fixed main container zIndex ([PR 2863](https://github.com/input-output-hk/daedalus/pull/2863))
 
+### Chores
+
+- Fixed Daedalus menu in Storybook used for theme and language selection ([PR 2886](https://github.com/input-output-hk/daedalus/pull/2886))
```

Comment:
Is it a chore or fix then? :D

Reply:
Not a fix as it doesn't touch the production code 😁